src/models/client/index.js

```diff
@@ -546,11 +543,15 @@ Client.createAndSendRentInvoices = async function(month = required()) {
 
   // .buildCheckinFormUrl .buildCheckoutFormUrl
   Client[`build${capType}FormUrl`] = async function(clientId) {
-    const [client, renting, packLevel] = await Promise.all([
-      Client.findById(clientId),
-      models.Renting.scope('latestRentingByClient').findOne({
+    const rentingScope =
+      `${type === 'checkout' ? 'current' : 'latest' }RentingByClient`;
+    const [[renting], packLevel] = await Promise.all([
+      models.Renting.scope(rentingScope).findAll({
```

Comment:
```suggestion
      models.Renting.scope(rentingScope).findOne({
```
J'ai l'impression que tu n'en veut qu'un seul ici, non ?